modules/core/src/main/java/org/apache/ignite/mxbean/IgnitionMXBean.java

```diff
@@ -118,13 +105,10 @@
      * @see org.apache.ignite.Ignition#stopAll(boolean)
      */
     @MXBeanDescription("Stops all started grids.")
-    @MXBeanParametersNames(
-        "cancel"
-    )
-    @MXBeanParametersDescriptions(
-        "If true then all jobs currently executing on all grids will be cancelled."
-    )
-    public void stopAll(boolean cancel);
+    public void stopAll(
+        @MXBeanParameterInfo(name = "cancel",
+            description = "If true then all jobs currently executing on all grids will be cancelled.") boolean cancel
+    );
```

Comment:
Could you please use new annotations for `restart()` method also?

Reply:
Well, yes, I can, if you'll tell me which of two names and descriptions use. 
I mean:
![image](https://user-images.githubusercontent.com/58294855/74640067-0b91f500-5180-11ea-8ef0-81a4c938814b.png)

Reply:
@l4ndsc4pe I believe is `cancel` parameter and the first description. Don't worry, I'll probably fix it on merge stage.